content/es/guides/directory-structure/assets.md

```diff
@@ -0,0 +1,286 @@
+---
+title: recursos
+description: El directorio de `assets` (recursos), contiene recursos sin compilar, como imágenes, fuentes de texto o archivos de Sass y Stylus.
+position: 2
+category: directory-structure
+csb_link: https://codesandbox.io/embed/github/nuxt-academy/guides-examples/tree/master/04_directory_structure/02_assets?fontsize=14&hidenavigation=1&theme=dark
+videoScript:
+  - assets-video.md
+questions:
+  - question: Cuál directorio contien los archivos sin compilar, Stylus o Sass, imágenes, o fuentes?
+    answers:
+      - static
+      - assets
+      - pages
+    correctAnswer: assets
+  - question: Cómo puedes referenciar el directorio de recursos (`assets`) dentro de los Vue templates?
+    answers:
+      - '/assets/tu_imagen.png'
+      - '@assets/tu_imagen.png'
+      - '@/assets/tu_imagen.png'
+    correctAnswer: '@/assets/tu_imagen.png'
+  - question: Cómo puedes referenciar el directorio de recursos dentro (`assets`) de archivos CSS?
+    answers:
+      - url("@assets/banner.svg")
+      - url("assets/banner.svg")
+      - url("@/assets/banner.svg")
+    correctAnswer: url("@assets/banner.svg")
+  - question: Dónde se incluyen estilos css globales?
+    answers:
+      - En tu archivo index.vue
+      - En el archivo nuxt.config.js
+      - El el archivo de `layout` defecto
```

Comment:
Change the first **_El_** for **_En_**. Additionally if you consider it more suitable, add the word _por_ after _layout_, so it reads _por defecto_ which I think is a good translation for _default_.

Reply:
- [ ] Change

Reply:
This sounds better